### ISSUE TITLE
Add unique IDs to H3s on Montgomery GI Bill Active Duty rates pages

### ISSUE THREAD

**Issue by RLHecht** [department-of-veterans-affairs/va.gov-team#50372]
## Issue Description

The CMS team accessibility specialist flagged that we have duplicative IDs on H3s on the [Montgomery GI Bill Active Duty rates](https://www.va.gov/education/benefit-rates/montgomery-active-duty-rates/) pages for both current and past rates, which is an issue for screen readers. Drupal will automatically create unique IDs if 2 headers are the same but because we copy and pasted this content within drupal, the IDs weren't updated.

[Slack thread for background](https://dsva.slack.com/archives/C01K37HRUAH/p1670249085485899)

When we update the IDs we should add a "-1" to the end of the tag to distinguish.

## Tasks
- [x] Update IDs (Megan)
- [x] Publish pages (Megan)
- [x] Notify CMS team in slack so they can confirm the issue is resolved

## Acceptance Criteria
- [x] Live pages have the correct distinct IDs

**Comment by megzehn** [comment]
Confirming that I adjusted the header tags and flagged it for Laura on Slack. CC @RLHecht.

**Comment by RLHecht** [comment]
This work is complete. Closing ticket!
